puppetlabs_spec_helper.gemspec

```diff
@@ -30,10 +30,10 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency 'mocha', '~> 1.0'
   spec.add_runtime_dependency 'pathspec', '>= 0.2', '< 2.0.0'
-  spec.add_runtime_dependency 'puppet-lint', '>= 2.5.2', '< 4.0.0'
+  spec.add_runtime_dependency 'puppet-lint', '>= 3.0.0', '~> 4.0.0.rc'
```

Comment:
I kind of wonder what this does. Isn't the `>= 3.0.0` part redundant? Same be below for `>= 2.0`.